diff --git a/node_modules/react-native-vision-camera/android/build.gradle b/node_modules/react-native-vision-camera/android/build.gradle
index 7d6d9a1..c4d4b1f 100644
--- a/node_modules/react-native-vision-camera/android/build.gradle
+++ b/node_modules/react-native-vision-camera/android/build.gradle
@@ -136,6 +136,12 @@ android {
     targetCompatibility JavaVersion.VERSION_1_8
   }
 
+  kotlinOptions {
+    freeCompilerArgs += [
+      "-opt-in=com.facebook.react.common.annotations.UnstableReactNativeAPI"
+    ]
+  }
+
   externalNativeBuild {
     cmake {
       path "CMakeLists.txt"
